src/Domain.LinnApps/PurchaseOrders/PurchaseOrderDelivery.cs

```diff
@@ -6,7 +6,7 @@ public class PurchaseOrderDelivery
     {
         public string Cancelled { get; set; }
 
-        public DateTime? DateAdvised { get; set; }
+        public DateTime? AdvisedDate { get; set; }
```

Comment:
I kinda prefer dateAdvised for consistency with other DateBlahs but not a biggie

Reply:
I can't recall why I changed it.. Possibly because that's what it's called on the db table

Reply:
Yeah I'm pretty sure it is called that on the table, maybe it was for consistency with mini Order Delivery etc if you'd called them AdvisedDate? I don't really mind either way as we know what it is both ways round